CHANGELOG.md

```diff
@@ -435,6 +435,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [14.2.1] - 2023-10-10
 ### Added
+- Added option to run GEOS-Chem at 0.125x0.15625 resolution using GEOS-FP derived winds fields generated from c720 mass fluxes archived by GMAO
```

Comment:
I suggest updating this to read "GEOS-Chem Classic" and specify it is for running nested simulations only. The PR description should be clear about this to to avoid confusion.

Reply:
I updated the name of the PR accordingly.  @msulprizio: can you change this in the CHANGELOG.md?

Reply:
Thanks for catching that. I modified that line in the merge commit [d74abd6](https://github.com/geoschem/geos-chem/commit/d74abd6d890195a867db22a6fc6a5f7e1210bf57).